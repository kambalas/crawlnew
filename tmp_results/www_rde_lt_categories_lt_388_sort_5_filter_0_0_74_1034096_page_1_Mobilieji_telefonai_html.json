[
  {
    "name": "iPhone 13 128GB Midnight",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Starlight",
    "price": "525.15",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Blue",
    "price": "598.65",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Pink",
    "price": "633.99",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Green",
    "price": "598.65",
    "error": false
  },
  {
    "name": "iPhone 13 256GB (PRODUCT)RED",
    "price": "806.01",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Blue",
    "price": "806.01",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Midnight",
    "price": "984.00",
    "error": false
  }
]