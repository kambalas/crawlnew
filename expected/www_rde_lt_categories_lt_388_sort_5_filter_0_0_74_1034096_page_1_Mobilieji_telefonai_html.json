[
  {
    "name": "iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Midnight",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE 4",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Midnight",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Starlight",
    "price": "425.59",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Pink",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Green",
    "price": "425.59",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Blue",
    "price": "514.96",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Pink Apple",
    "price": "523.96",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Green Apple",
    "price": "494.75",
    "error": false
  },
  {
    "name": "iPhone 13 128GB Blue Apple",
    "price": "494.75",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Midnight",
    "price": "666.12",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Starlight",
    "price": "666.12",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Blue",
    "price": "666.12",
    "error": false
  },
  {
    "name": "iPhone 13 256GB (PRODUCT)RED",
    "price": "666.12",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Pink",
    "price": "666.12",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Green",
    "price": "598.75",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Starlight",
    "price": "598.75",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Blue",
    "price": "598.75",
    "error": false
  },
  {
    "name": "iPhone 13 256GB (PRODUCT)RED",
    "price": "598.75",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Pink Apple",
    "price": "666.12",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Green Apple",
    "price": "598.75",
    "error": false
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  }
]